### ISSUE TITLE
AssertionError: Sample size has to be bigger than the patch size

### ISSUE THREAD

**Issue by lorenzocerrone** [kreshuklab/panseg#146]
@wolny This error happens if the image is smaller that the patch size in any give dimension. I think this can be avoided by checking it in the `@widget_unet_predictions.model_name.changed.connect`.  Do you agree?

**Comment by wolny** [comment]
yes, that should be doable since we have access to the `widget_iterative_unet_predictions.image` in there, so we could change the patch size accordingly. But in addition, there should be an additional check of the image and patch_size somewhere in the prediction code
